packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4333,6 +4342,21 @@ export default class Meeting extends StatelessWebexPlugin {
       data: {trigger: trigger.USER_INTERACTION, isRoapCallEnabled: true},
     });
 
+    if (!isEmpty(this.meetingInfo)) {
+      Metrics.postEvent({
+        event: eventType.MEETING_INFO_REQUEST,
```

Comment:
just to double check: meetingLookupUrl is not required in this CA event and only required in the event for the response?

Reply:
yes, that's correct